openhtf/util/units/bin/units_from_xls.py

```diff
@@ -0,0 +1,160 @@
+# coding: iso-8859-1
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Read in a .xls file and generate OpenHTF Unit objects.
+
+UNECE, the United Nations Economic Commision for Europe, publishes a set of
+unit codes for international trade in the form of Excel spreadsheets (.xls
+files) here:
+
+http://www.unece.org/cefact/codesfortrade/codes_index.html
```

Comment:
Mention the xls is in the zip file at the bottom under "Codes for Units of Measurement used in the International Trade". This wasn't obvious to me months ago when I got to that page